models/src/local/ColorControlOverrides.ts

```diff
@@ -47,6 +87,60 @@ LocalMatter.children.push({
         },
 
         // Spec states the values of this bitmap are the same as the feature map
-        { tag: "attribute", id: 0x400a, name: "ColorCapabilities", type: "FeatureMap" },
```

Comment:
Why'd you have to do this?  Was there an issue using the FeatureMap enum?

Reply:
Yes. I could not assigning to this.features because is a bit different in structure type wise.